tests/unit/backend/recipe/algorithm/test_PixelGroupingParametersCalculationAlgorithm.py

```diff
@@ -341,7 +341,7 @@ def test_local_wrong_idf():
                 groupingFile=groupingFile,
                 referenceParametersFile=referenceParametersFile,
             )
-        assert "File 'junk' does not exist" in str(excinfo.value)
```

Comment:
@darshdinger this line here suddenly started causing problems for me locally this morning.  This test checks the error being returned  by checking the error string.  For some reason, this morning, with no changes made to the algo, this stared throwing a different error message when run locally.  This change made the test pass locally.  Not sure what happens elsewhere.